apps/civicsignalblog/src/payload/components/actions/index.tsx

```diff
@@ -0,0 +1,98 @@
+import { useAuth } from "payload/components/utilities";
+import React, { useEffect, useState } from "react";
+
+import applications from "../../../lib/data/json/applications";
+
+function BeforeDashboard() {
+  const { user } = useAuth();
+
+  const [selectedApp, setSelectedApp] = useState(
+    String(user.currentApp ?? user.defaultApp) ?? "",
```

Comment:
```suggestion
    user.currentApp || user.defaultApp || RESEARCH,
```

I'd rather we use really defaults whenever we can instead of `""`. Alternatively, ensure user.defaultApp is never undefined/null/empty.